test/e2e/spec/e2e_spec.rb

```diff
@@ -113,9 +113,13 @@ def run_contract(contract_setup, scripts)
       policy_id = get_policy_id(policy)
       mint_script = "mintBurn_1.json"
       burn_script = "mintBurn_2.json"
-      assets = [{ "asset_name" => asset_name("mint-burn"),
-                 "quantity" => 1,
-                 "policy_id" => policy_id }]
+      assets = [ {"policy_script" => {"language_version" => "v1", "script_type" => "plutus"},
+                  "policy_id" => policy_id,
+                  "assets" => [ {"fingerprint" => "asset1q78ea9ds0rc3tfwu2damsjehjup2xuzddtg6xh",
+                                 "amount" => {"quantity" => 1, "unit" => "assets"},
```

Comment:
I was wondering if this cannot be reduced to:
```
"quantity" => 1,
```
?

That would be in line with what we have shown in wallet balance:
```
  "assets": {
    "total": [
      {
        "asset_name": "",
        "quantity": 1,
        "policy_id": "817dae91a171fb9fc3e936ff065a68c3e8ca48fbf28302fb789976ca"
      },
      {
        "asset_name": "1111",
        "quantity": 1,
        "policy_id": "817dae91a171fb9fc3e936ff065a68c3e8ca48fbf28302fb789976ca"
      }
```
_maybe that's not in scope of this PR though_

Reply:
sounds reasonable. will do it in coming commit